multimodal/image_question_answering/app.py

```
@@ -0,0 +1,195 @@
+
+    _page_images = pdf_to_images(pdf_file.name)
+    n = len(_page_images)
+
+    return (
+        _page_images[0],
+        gr.update(
+            minimum=1,
+            maximum=n,
+            value=1,
+            step=1,
+            label=f"Page (1–{n})",
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)